### ISSUE TITLE
[DOCS] Coordinate information is deleted from the results after the Inference operation

### ISSUE THREAD

**Issue by williamobrein** [CosmiQ/solaris#292]
_Thank you for helping to improve `solaris`!_

## Documentation request summary

1. The coordinate information is deleted from the results after the Inference operation.
2. Config file metric field.

## Task detail and notes

1. Details
When I examined the results after performing an inference, I noticed that the coordinate information was lost. Do you have any suggestions on this topic? Why is it disappearing? I didn't do anything specifically to erase this coordinate information. I did the training and inference with the automatic machine learning command you specified in your documentation.
To explain myself better. For example, if I say to divide 512x512 pieces, pixel information is coming instead of coordinate information. This prevents me from drawing the polygons on the original tif. If there is any information in this documentation, please do not hesitate. If you don't, I'd appreciate it if you could help.

2. Details
You have placed the metric option in the configuration file for the training and valid fields. It should be in list format as far as I have examined in the documentation. But you have not provided information about the spelling format of metrics. For example, ['f1', 'acc'] and so on. Can you give me an example of how I should edit the metric field?

**Comment by nrweir** [comment]
Hi @williamobrein,

The problem is that neural networks have no idea how to handle coordinate reference systems - predictions have to be made in pixel coordinates and then converted. Have a look at `sol.vector.mask.mask_to_poly_geojson()` in [the tutorials](https://solaris.readthedocs.io/en/latest/tutorials/notebooks/api_mask_to_vector.html) and [the API docs](https://solaris.readthedocs.io/en/latest/api/vector.html#solaris.vector.mask.mask_to_poly_geojson).

**Comment by williamobrein** [comment]
Hi, I solved the error when you debug your code. You have arranged the `mask_to_poly_geojson ()` part according to the projection of the Spacenet data. Because the projection was different, the value of the `min_area` should have changed. Thanks!

You didn't say anything about metric? What exactly is the format? I didn't realize.

**Comment by nrweir** [comment]
Metrics are still under active development. At present, they can only be provided for Keras models using the standard Keras naming scheme; it's something we're planning to add for Torch when we get to it (#175)

**Comment by williamobrein** [comment]
Thanks!
